Add "Lotus install" section. (#235)
filecoin-project/filecoin-docs: docs/install/lotus/requirements.md

---
title: Requirements
description: There are specific system and hardware requirements for running a Lotus node or mining set up.
---

# Requirements

The requirements of your system will change depending on the type of client you are running: a node, a storage miner, or seal worker.

## Hardware

### Lotus node

> Since the Filecoin network is constantly growing and being developed, hardware requirements keep changing. The requirements table below is correct as of writing but may change in the future. The amount of memory (RAM) required is expected to drop significantly as the Lotus client uses Badger FS more efficiently. This is expected to come down to about 1GB in the near future.

Reviewer: ```suggestion
The requirements table below is correct as of August 2020 but may change in the future as software develop continues. The amount of memory (RAM) required is expected to drop significantly to about 1GB in the future as the Lotus client uses Badger FS more efficiently. 
```

| Processor                                      | Memory   | Storage space         |
| ---------------------------------------------- | -------- | --------------------- |
| 4 cores<br>8 threads<br> 2.5GHz base frequency | 8GB DDR4 | 200GB (SSD preferred) |

> Again, these requirements will change in the near future.

Reviewer: ```suggestion
```

Reviewer: Already mentioned 2x earlier :)

### Storage miner

> Miners require a significant amount of hardware to run. The setup below is a minimal example for sealing 32GB sectors on Lotus:

Reviewer: ```suggestion
Miners require a significant amount of hardware to run. The setup below is a minimal example for sealing 32GiB sectors on Lotus:
```

because [sectors are measured in binary](https://docs.filecoin.io/how-to/store/large-files/#maximizing-storage)
